source/change-streams/tests/README.rst

```diff
@@ -155,3 +155,8 @@ The following tests have not yet been automated, but MUST still be tested
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
 #. ``ChangeStream`` will resume after a ``killCursors`` command is issued for its child cursor.
+#. ``ChangeStream`` will resume multiple times even when no intervening change stream documents are received by the cursor. Consider this scenario:
```

Comment:
Is it worth saying that this could be done as a unit test if possible, and manually otherwise?  I can't see a driver actually implementing this as an automated prose test, at least not until after the live failover testing infrastructure is in place.

Reply:
Can't we just test this with `failCommand` setting `getMore`  to fail >=2 times?

Reply:
Reading the steps in [SPEC-1073](https://jira.mongodb.org/browse/SPEC-1073) that lead to this change, I would think that multiple `getMore` failures could trigger this. I believe the current change stream tests don't allow for interleaving insert operations during change stream iteration, but perhaps we can simulate shorter `getMore` responses by using batchSize to only request a single change record at a time.

Reply:
So it would be 

# aggregate (succeeds)
# getMore (fails with retryable error)
# aggregate (succeed)
# getMore (fails with retryable error)
# aggregate (succeeds)
# getMore (succeeds)
# change stream document received (not sure how to do this un-racily)

Reply:
Is there a race condition if you insert all of the documents up front and start the initial change stream with the appropriate operation time indicating that you want to see all of those inserts? With a small batchSize, you could then successfully receive them despite the two `getMore` failures.

Reply:
Done